example/demo.py

```diff
@@ -27,12 +29,22 @@
     viz.video(tensor=video)
 
     # video demo: download video from http://media.w3.org/2010/05/sintel/trailer.ogv
-    videofile = '/home/%s/trailer.ogv' % getpass.getuser()
+    video_url = 'http://media.w3.org/2010/05/sintel/trailer.ogv'
+    # linux
+    if _platform == "linux" or _platform == "linux2":
+        videofile = '/home/%s/trailer.ogv' % getpass.getuser()
+    # MAC OS X
+    elif _platform == "darwin":
+        videofile = '/Users/%s/trailer.ogv' % getpass.getuser()
+    # download video
+    urllib.request.urlretrieve(video_url, videofile)
```

Comment:
This will work in Python 3 but break in Python 2 (there is no `urllib.request` in Python 2).